tools/encodings/GF Glyph Sets/GF-latin-pro_optional-glyphs.nam

```diff
@@ -0,0 +1,37 @@
+          at.case
+          hyphen.case
+          endash.case
+          emdash.case
+          periodcentered.case
+          bullet.case
+          slash.case
+          backslash.case
+          parenright.case
+          bracketleft.case
+          bracketright.case
+          braceleft.case
+          braceright.case
+          guilsinglleft.case
+          guilsinglright.case
+          guillemetleft.case
+          guillemetright.case
+          0x0030 zero.sinf
```

Comment:
In https://github.com/google/fonts/issues/675#issuecomment-282672569 we found out that Namelists are parents and filter-lists are derrived from Namelists. In #678 I started writing tests for that. These changes are making the tests pass.

The rule is: if it is in a filter-list it must be also in the according Namelist.

I think we have some room for more advanced rules here. We could demand that it must be either in the `unique` or in the `optional` Namelist. We could also add another optional type of namelist to store components for construction.